app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file_using_cp_command(command: str) -> None:
+    command_parsed = command.split(" ")
+    src_file_name = command_parsed[1]
+    dst_file_name = command_parsed[2]
+
+    if src_file_name == dst_file_name:
+        print(f"File wasn't copied: source file '{src_file_name}' "
+              f"has the same name as the destination file.")
+    else:
+        with open(src_file_name, "r") as src_file, \
+             open(dst_file_name, "w") as dst_file:
+
+            file_in_data = src_file.readline()
+            dst_file.writelines(file_in_data)
```

Comment:
The way you're copying the file content is not efficient. You're only copying the first line of the source file. To copy the whole file, you should read it line by line in a loop, or better yet, use the shutil.copyfile() function from the shutil module in Python, which is designed for this exact purpose.